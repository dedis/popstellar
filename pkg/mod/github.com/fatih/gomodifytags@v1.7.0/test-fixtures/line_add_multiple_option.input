package foo

type foo struct {
	bar    string `json:"bar"`
	f      bool
	t      bool
	Ankara []string `json:"ankara"`
	a      []string `json:"a"`
}
