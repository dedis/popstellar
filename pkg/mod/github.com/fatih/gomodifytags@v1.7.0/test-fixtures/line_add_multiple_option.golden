package foo

type foo struct {
	bar    string   `json:"bar,omitempty" hcl:"bar,squash"`
	f      bool     `hcl:"f,squash"`
	t      bool     `hcl:"t,squash"`
	Ankara []string `json:"ankara,omitempty" hcl:"ankara,squash"`
	a      []string `json:"a"`
}
